diff --git a/node_modules/readable-stream/lib/_stream_writable.js b/node_modules/readable-stream/lib/_stream_writable.js
index 4bdaa4f..8b1ed26 100644
--- a/node_modules/readable-stream/lib/_stream_writable.js
+++ b/node_modules/readable-stream/lib/_stream_writable.js
@@ -38,6 +38,7 @@ util.inherits = require('inherits');
 /*</replacement>*/
 
 var Stream = require('stream');
+var process = require("process");
 
 util.inherits(Writable, Stream);
 
